### PR TITLE
set app as owner of log folder in install/restore/update script

### PR DESCRIPTION
**Pull request by xoddark**
This fix problem of writing log from php actualize_script runed by cron. Issue #148

## Problem

- *Description of why you made this PR*

## Solution

- *Set app user as owner of app log folder*

## PR Status

- [X] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)
I have tested the manual change of the owner was tested on server, but not the change of the install/restore/update script.

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by plopoyop** [comment]
!testme

**Comment by plopoyop** [comment]
Install fail : 
```152217 WARNING ./install: line 148: log_path: command not found
153734 ERROR Unable to install freshrss: An error occurred inside the app installation script ```

You must remove spaces when affecting defining variable in bash.
You should try to manually install the package to tests you changes

File: scripts/install [hunk 1]
```diff
@@ -145,7 +145,9 @@ chmod 644 "/etc/cron.d/$app"
 #=================================================
 ynh_script_progression --message="Configuring log rotation..." --weight=1
 
-mkdir -p "/var/log/$app"
+log_path = "/var/log/$app"
```
Comment (plopoyop) [inline scripts/install:148]: should be
```
log_path = "/var/log/$app"
```

File: scripts/upgrade [hunk 2]
```diff
@@ -167,7 +167,9 @@ fi
 #=================================================
 ynh_script_progression --message="Upgrading logrotate configuration..." --weight=1
 
-mkdir -p "/var/log/$app"
+log_path = "/var/log/$app"
```
Comment (plopoyop) [inline scripts/upgrade:170]: should be
```
log_path = "/var/log/$app"
```

File: scripts/install [hunk 3]
```diff
@@ -145,7 +145,9 @@ chmod 644 "/etc/cron.d/$app"
...
+mkdir -p $log_path
+chown $app:www-data log_path
```
Comment (plopoyop) [inline scripts/install:150]: missing `$`

File: scripts/restore [hunk 4]
```diff
@@ -108,7 +108,9 @@ ynh_restore_file --origin_path="/etc/cron.d/$app"
 #=================================================
 ynh_script_progression --message="Restoring the logrotate configuration..." --weight=1
 
-mkdir -p "/var/log/$app"
+log_path = "/var/log/$app"
+mkdir -p $log_path
+chown $app:www-data log_path
```
Comment (plopoyop) [inline scripts/restore:113]: missing `$`

**Comment by xoddark** [comment]
@plopoyop : thank you for your feedback.
I do it rapidly, as a start for a fix ;)

**Comment by plopoyop** [comment]
All ok.
Thanks for your contribution :)

**Pull request merged**